@ecomplus/storefront-components/src/js/ProductKitVariations.js

```diff
@@ -0,0 +1,223 @@
+import {
+  i19addToCart,
+  i19close,
+  i19next,
+  i19previous,
+  i19quantity,
+  i19selectVariationMsg,
+  i19item,
+  i19minQuantity,
+  i19maxQuantity
+} from '@ecomplus/i18n'
+
+import {
+  i18n,
+  name as getName,
+  img as getImg,
+  randomObjectId as genRandomObjectId
+} from '@ecomplus/utils'
+
+import ecomCart from '@ecomplus/shopping-cart'
+import Glide from '@glidejs/glide'
+import APicture from '../APicture.vue'
+import ProductVariations from '../ProductVariations.vue'
+import APrices from '../APrices.vue'
+import ALink from '../ALink.vue'
+import AAlert from '../AAlert.vue'
+
+export default {
+  name: 'ProductKitVariations',
+
+  components: {
+    ALink,
+    AAlert,
+    APicture,
+    APrices,
+    ProductVariations
+  },
+
+  props: {
+    items: {
+      type: Array,
+      required: true
+    },
+    min: {
+      type: Number,
+      default: 1
+    },
+    max: Number,
+    maxOptionsBtns: {
+      type: Number,
+      default: 10
+    },
+    slug: String,
+    buyText: String,
+    kitProductId: String,
+    kitName: String,
+    kitPrice: Number,
+    canAddToCart: {
+      type: Boolean,
+      default: true
+    },
+    hasBuyButton: {
+      type: Boolean,
+      default: true
+    },
+    glideOptions: {
+      type: Object,
+      default () {
+        return {
+          type: 'slider',
+          autoplay: false,
+          rewind: false
+        }
+      }
+    },
+    isSSR: Boolean
```

Comment:
```suggestion
    }
```

Você não usa isso, aliás não vejo como isso poderia ser usado nesse componente...